Frameworks/AddressBook/ABRecord.mm

```diff
@@ -47,12 +83,117 @@ bool ABRecordSetValue(ABRecordRef record, ABPropertyID property, CFTypeRef value
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Caveat
+ @Notes Properties that return a Multi-Value are not yet supported, and a few properties
+        (alternate birthday, creation/modification dates, person kind, phonetic middle name)
+        don't have direct equivalents.
 */
 CFTypeRef ABRecordCopyValue(ABRecordRef record, ABPropertyID property) {
```

Comment:
Tried doing a switch, but since the kAB... constants are extern, the compiler complains about the case expressions not being constant.

---
In reply to: [74811019](https://github.com/Microsoft/WinObjC/pull/881#discussion_r74811019) [](ancestors = 74811019)